src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -88,6 +88,65 @@
           </div>
         </div>
 
+        <!-- Mirroring -->
+        <div class="form-group row">
+          <div class="cd-col-form-offset">
+            <div class="custom-control custom-checkbox">
+              <input type="checkbox"
+                     class="custom-control-input"
+                     id="mirroring"
+                     name="mirroring"
+                     (change)="setMirrorMode()"
+                     [(ngModel)]="mirroring && this.currentPoolName"
+                     formControlName="mirroring">
+              <label class="custom-control-label"
+                     for="mirroring">Mirroring</label>
+              <cd-help-text *ngIf="currentPoolMirrorMode === 'disabled'">
```

Comment:
Add `cd-help-text` for mirroring, Journal and Snapshot.

Mirroring: `Allow data to be asynchronously mirrored between two Ceph clusters`
Journal/snapshot: https://docs.ceph.com/en/reef/rbd/rbd-mirroring/#rbd-mirroring